EIPS/7716.md

```diff
@@ -0,0 +1,84 @@
+---
+eip: 7716
```

Comment:
@dapplion @nerolation can you rename the file to eip-7716.md

Reply:
ups, fixed